apps/cli/config/config.exs

```diff
@@ -2,29 +2,18 @@
 # and its dependencies with the aid of the Mix.Config module.
 use Mix.Config
 
-# This configuration is loaded before any dependency and is restricted
-# to this project. If another project depends on this project, this
-# file won't be loaded nor affect the parent project. For this reason,
-# if you want to provide default values for your application for
-# 3rd-party users, it should be done in your "mix.exs" file.
+config :cli,
+  cookie:
+    :erlang.apply(
+      fn ->
+        cookie =
+          if String.contains?(System.cwd!(), "apps") do
+            String.trim(File.read!(Enum.join(["../../", "COOKIE"])))
```

Comment:
I think you could use `Path.join` instead of `Enum.join` here. Not sure if it would make a big difference, but it might make more sense since we're dealing with paths?